checkstyle-tester/launch.groovy

```diff
@@ -207,13 +206,20 @@ def runMavenExecution(srcDir, excludes, checkstyleConfig) {
     println "Running Checkstyle on $srcDir - finished"
 }
 
-def postProcessCheckstyleReport(siteDir) {
+def postProcessCheckstyleReport(targetDir) {
+    def siteDir = "$targetDir/site"
     println 'linking report to index.html'
     new File("$siteDir/index.html").renameTo "$siteDir/_index.html"
     Files.createLink(Paths.get("$siteDir/index.html"), Paths.get("$siteDir/checkstyle.html"))
 
     removeNonReferencedXrefFiles(siteDir)
     removeEmptyDirectories(new File("$siteDir/xref"))
+
+    new AntBuilder().replace(
+        file: "$targetDir/checkstyle-result.xml",
+        token: "src/main/java",
```

Comment:
I don't know this replace method, but what happens if a project we are regressing on has a directory called `src/main/java`?
Lets take this as example: `/home/rnveach/checkstyle-tester/src/main/java/my_project_abcdef/src/main/java/...`
Won't it replace it to be `/home/rnveach/checkstyle-tester/repositories/my_project_abcdef/repositories/...` instead of `/home/rnveach/checkstyle-tester/repositories/my_project_abcdef/src/main/java/...`? The second should be the correct final form, not the first.
If we are to do replaces like this, we should probably be using the fully qualified path.